database/init.js

```diff
@@ -0,0 +1,77 @@
+'use strict'
+
+const _ = require('lodash')
+const async = require('async')
+const pg = require('pg')
+const config = require('./../src/lib/config')
+
+if (config.get('env') !== 'test') {
+  console.log('ERROR: You are trying to init the database while not in test environment.')
+  process.exit(1)
+}
+
+const pgConf = _.clone(config.get('pgdb'))
+/**
+ * This is a hack to connect to PostgreSQL if you do not have a specific db.
+ * @see https://github.com/olalonde/pgtools/blob/master/index.js#L43
+ */
+pgConf.database = 'postgres'
```

Comment:
@paolochiodi 

> `pgConf.database = pgConf.database || 'postgres' `

it will no work. `pg` needs a database to connect to and here we are trying to reset it or create it if it does not exists :)